sdk/core/azure-core/tests/test_streaming.py

```diff
@@ -92,6 +93,21 @@ def test_decompress_compressed_no_header(http_request):
     except UnicodeDecodeError:
         pass
 
+@pytest.mark.parametrize("http_request", HTTP_REQUESTS)
+def test_compress_compressed_no_header_offline(port, http_request):
+    # expect compressed text
+    request = http_request(method="GET", url="http://localhost:{}/streams/compressed_no_header".format(port))
+    with RequestsTransport() as sender:
+        try:
+            response = sender.send(request, stream=True)
```

Comment:
@marcusc2 can you use `pipeline.run` instead of `send`? For our newer azure core rest, everything needs to be properly read, so easier to use `pipeline.run` instead.

```python
client = PipelineClient("")
request = http_request(method="GET", url="http://localhost:{}/streams/compressed_no_header".format(port))
pipeline_response = client._pipeline.run(request, stream=True)
```